docs/administration/configure-proxy-cache/_index.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Configure Proxy Cache
+weight: 35
+---
+
+Proxy cache allows you to use Harbor to proxy and cache images from a target public or private registry. You can use a proxy cache to pull images from a target Harbor or non-Harbor registry in an environment with limited or no access to the internet. You can also use a proxy cache to limit the amount of requests made to a public registry, avoiding consuming too much bandwidth or being throttled by the registry server.
+
+A Harbor system administrator configures a proxy cache by creating a proxy cache project. Proxy cache projects connect to a target registry using a registry endpoint you configure. Harbor only supports Docker Hub and Harbor registry endpoints for proxy caching. See how to [Create an Endpoint](../configuring-replication/create-replication-endpoints.md).
```

Comment:
maybe move 'Harbor only supports Docker Hub and Harbor registry endpoints for proxy caching' to before 'proxy cache projects connect to a target...'

Reply:
I think we should also make it clear that user should now change his docker pull command and his pod manifest to hit that harbor proxy cache instead of the original target registry.  Just to make it a bit clearer